kitsune/sumo/static/sumo/scss/components/_card.scss

```diff
@@ -181,21 +181,79 @@
   }
 
   &--topic {
-    @include c.elevation-01;
     display: flex;
-    align-items: center;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+    background-color: #fff;
+    border: 1px solid #ddd;
+    border-radius: 8px;
```

Comment:
Let's use [design tokens](https://protocol.mozilla.org/docs/fundamentals/design-tokens) from protocol for consistency with the rest of the codebase.

Reply:
Got it.